db/version_edit_handler.cc

```diff
@@ -909,7 +909,15 @@ void DumpManifestHandler::CheckIterationResult(const log::Reader& reader,
       fprintf(stdout, "comparator: %s\n", cfd->user_comparator()->Name());
     }
     assert(cfd->current());
-    fprintf(stdout, "%s \n", cfd->current()->DebugString(hex_).c_str());
+
+    // Print out DebugStrings. Null characters are replaced with whitespaces.
```

Comment:
Hmm. If I redirect the output to a file and open it in a hex editor, I expect to see the raw data, not null bytes replaced with spaces.